### PR TITLE
docs: add glossary.md

### LINKED ISSUE #557: Suggestion: Add glossary.md to tips folder
# Add glossary.md to [tips](https://github.com/EddieJaoudeCommunity/support/tree/main/tips) folder 
After I watched this [Video](https://www.youtube.com/watch?v=4Zc9ci9L5wY) by Fireship.io, I thought that it is a good idea for my first OpenSource contribution.
A markdown file that contains a list of Programing terms with explanation and code examples if needed, and some additional resources to learn more about it.

### Anonymous Function
An anonymous function is a function that was declared without any named identifier to refer to it.

#### Code Examples
```javascript
window.addEventListener("click", function () {
    // This is An Anonymous Function
})
``` 

#### Additional Resources
https://en.wikipedia.org/wiki/Anonymous_function

### PR DESCRIPTION
**Pull request by ayoubmehd**
closes #557 

Add glossary.md

- Add glossary.md file to the tips Folder
- Add an Example to glossary.md File

We descuss this on discord and github in this [issue](https://github.com/EddieHubCommunity/support/issues/557)

Is this good as my first real contributon to open source 🥺?

### CODE REVIEW

**Review by schmelto — APPROVED** [review]

File: tips/glossary.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Glossary
+
+You will find a bunch of useful programming terms that you must know.
+
+## Anonymous Function
+
+It a function declared without any named identifier to refer to it
```
Comment (BOLT04) [inline tips/glossary.md:7]: how about we change this?
```suggestion
It's a function declared without any named identifier to refer to it.
```

**Review by BOLT04 — COMMENTED** [review]
Thank you for your contribution @ayoubmehd 👍, left just one comment

File: tips/glossary.md (hunk 1 above)
Comment (ayoubmehd) [reply tips/glossary.md]: @BOLT04 Thanks

**Review by Vyvy-vi — APPROVED** [review]
LGTM-!
However, in future, maybe we could use tables for this so that the file looks neater

**Review by BOLT04 — APPROVED** [review]
Thank you @ayoubmehd 👍

**Pull request merged**